examples/vm_scheduling/rule_based_algorithm/README.md

```diff
@@ -0,0 +1,29 @@
+# Rule Based Algorithm
+
+## Overview
+
+The VM scheduling problem is one of the quintessential use cases of MARO. The rule based algorithm can be run with a set of scenario configurations that can be found under maro/simulator/scenarios/vm_scheduling. General experimental parameters (e.g., type of topology, type of algorithm to use) can be configured through `config.yml`.
+
+## Algorithms
+
+- **Random Pick**: Random pick an available PM.
```

Comment:
when a VM request is coming, randomly choose an available PM with enough resources to place the VM.

mode detailed like this.
similar to other algorithms